packages/config/config/devices/0x031e/lzw60.json

```diff
@@ -110,7 +157,7 @@
 		},
 		"102": {
 			"label": "Humidity Reporting Interval",
-			"description": "Note: The set value will round to the nearest minute. (0 = Off)",
+			"description": "Interval, in seconds, in which humidity reports should be sent. 0=Disabled. Note: The set value will round to the nearest minute.",
```

Comment:
I don't like this change. We're trying to keep these short and not have redundancy with the label and the definitions (unit, etc.) below. Renaming "Off" to "Disabled" is okay IMO.

Reply:
Same goes for the others.